File: 1074.py
rep=int(input())
for i in range(rep):
    num=int(input())
    if num % 2 == 0 and num > 0:
        print("EVEN POSITIVE")
    elif num % 2 == 0 and num < 0:
        print("EVEN NEGATIVE")
    elif num % 2 != 0 and num > 0:
        print("ODD POSITIVE")
    elif num % 2 != 0 and num < 0:
        print("ODD NEGATIVE")
    elif num == 0:
        print("NULL")

File: cpp/1074.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    int n; cin >> n;
    
    for(int i=0; i<n; i++){
        int num; cin >> num;

        if(num == 0){
            cout << "NULL" << "\n";
        }
        else{
            if (num > 0 && num%2 == 0){
                cout << "EVEN POSITIVE" << "\n";
            }
            else if (num > 0 && num%2 != 0){
                cout << "ODD POSITIVE" << "\n";
            }
            else if (num < 0 && num%2 == 0){
                cout << "EVEN NEGATIVE" << "\n";
            }
            else{
                cout << "ODD NEGATIVE" << "\n";
            }
        }
    }

}